src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -38,6 +38,19 @@ public IWorkspaceModel CurrentWorkspaceModel
             }
         }
 
+        private ICommandExecutive commandExecutive;
+        public virtual ICommandExecutive CommandExecutive 
```

Comment:
Uhm... I'm not sure about the `virtual` part though, is there a reason it needs to be made `virtual`?